samples/spreadsheet/insert_a_new_worksheet/cs/Program.cs

```diff
@@ -1,27 +1,28 @@
-#nullable disable
-
 using DocumentFormat.OpenXml.Packaging;
 using DocumentFormat.OpenXml.Spreadsheet;
 using System.Linq;
 
+InsertWorksheet(args[0]);
+
 // Given a document name, inserts a new worksheet.
 static void InsertWorksheet(string docName)
 {
     // Open the document for editing.
     using (SpreadsheetDocument spreadSheet = SpreadsheetDocument.Open(docName, true))
     {
+        WorkbookPart workbookPart = spreadSheet.WorkbookPart ?? spreadSheet.AddWorkbookPart();
         // Add a blank WorksheetPart.
-        WorksheetPart newWorksheetPart = spreadSheet.WorkbookPart.AddNewPart<WorksheetPart>();
+        WorksheetPart newWorksheetPart = workbookPart.AddNewPart<WorksheetPart>();
         newWorksheetPart.Worksheet = new Worksheet(new SheetData());
 
-        Sheets sheets = spreadSheet.WorkbookPart.Workbook.GetFirstChild<Sheets>();
-        string relationshipId = spreadSheet.WorkbookPart.GetIdOfPart(newWorksheetPart);
+        Sheets sheets = workbookPart.Workbook.GetFirstChild<Sheets>() ?? workbookPart.Workbook.AppendChild(new Sheets());
```

Comment:
should we create a new API here for `GetFirstChildOrCreate<TPart>()`?

Reply:
That would be a more clear than `Sheets sheets = workbookPart.Workbook.GetFirstChild<Sheets>() ?? workbookPart.Workbook.AppendChild(new Sheets());`

Reply:
https://github.com/dotnet/Open-XML-SDK/issues/1548